content/300-accelerate/900-compare.mdx

```diff
@@ -0,0 +1,106 @@
+
+**Why are these important?**
+
+- If you decide to manage a connection pooler yourself (eg. using pgbouncer or pgcat) you will also be responsible for managing its uptime. If the server crashes, your application may be down until you recover it. Accelerate, as a fully managed solution will be recovered for you transparently, in the unlikely case of any infrastructure issue.
+- The hosted pgbouncer option on Digital Ocean is semi-managed, you will need to set it up in your Digital Ocean account, and ensure it is running smoothly at all times.
+- Authenticating with an API key can be a helpful security measure, allowing you to decouple database credentials from application secrets. Easily rotate API keys as often as you like, without needing any credential changes in your database
+- Redundancy is helpful in the unlikely scenario that your connection pool service goes down. With Accelerate, it is automatically and seamlessly handed over to another server and recovered without any interruption.
+
+### Performance
+
+|  | Accelerate | pgbouncer | pgcat | Digital Ocean (pgbouncer) | Neon (pgbouncer) | Supavisor | Hyperdrive |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| **Auto scaling** | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| **Globally distributed** | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ✅ |
+| **Optimized queries over HTTP** | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ✅ |
+| **Isolated compute** | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+
+**Why are these important?**
+
+- Accelerate will automatically scale up and down to suit your application workload, meaning you'll never run out of compute resource. Additionally, this provides important redundancy to protect against any single compute instance failing — in the unlikely event of an instance going down, Accelerate will automatically spawn a new instance.
+- Cross-region TCP handshakes between the application server and PgBouncer or the database are costly and time-consuming. If connections are reused only at the PgBouncer layer, the TCP handshake and connection setup still consume unnecessary time on every single request, which undermines the efficiency of connection reuse. Prisma Accelerate improves this by leveraging HTTP, which is more efficient for connection management. It reduces the overhead associated with TCP handshakes, resulting in faster, more responsive interactions between your application and the database.
+- Never worry about 'noisy neighbors' with isolated compute resources. Other customers never impact on your own performance.
+
+### Database Support
+
+|  | Accelerate | pgbouncer | pgcat | Digital Ocean (pgbouncer) | Neon (pgbouncer) | Supavisor | Hyperdrive |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| **PostgreSQL** | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| **MySQL** | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| **Planetscale** | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| **CockroachDB** | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| **MongoDB** | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| **MariaDB** | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
```

Comment:
I'd remove MariaDB, unless you think that people really really care about it coming soon

Are there no connection pool solutions for the other ones?

Reply:
@petradonka I suggested that we add MariaDB for transparency. 

> unless you think that people really really care about it coming soon

I recall some users attempting to try out Accelerate with it, @nurul3101 can elaborate regarding the demand for MariaDB.

Reply:
There are for sure connection pool solutions for other ones, but this page doesn't aim to be a full representation of everything, just the commonly perceived alternatives to Accelerate. 

I'd also say that we can add to this page as we learn more -- I'd err on the side of concise and pointy (easy & useful for readers) rather than cover everything on the market (overwhelming)

Reply:
Yep, I agree. So just remove MariaDB then, if you agree :)